src/main/java/io/appium/java_client/proxy/Helpers.java

```diff
@@ -27,13 +27,11 @@
 import net.bytebuddy.matcher.ElementMatcher;
 import net.bytebuddy.matcher.ElementMatchers;
 import org.jspecify.annotations.Nullable;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.remote.RemoteWebElement;
 
 import java.lang.reflect.Method;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Map;
-import java.util.Set;
-import java.util.WeakHashMap;
+import java.util.*;
```

Comment:
Please avoid using star imports

Reply:
Do you have any code style I can import? I looked in the project but could not find any.